src/app/(home)/_components/home-feed.tsx

```diff
@@ -46,6 +55,7 @@ const getHomeFeed = unstable_cache(
 
 export async function HomeFeed({ page }: HomeFeedProps) {
   const homeFeed = await getHomeFeed(page);
+  const totalPages = await getTotalPages();
```

Comment:
We can use Promise.all() here to run getHomeFeed() and getTotalPages() concurrently